src/TimeoutMigrationTool/RabbitMq/RabbitMqTimeoutTarget.cs

```diff
@@ -95,7 +96,7 @@ ValueTask<MigrationCheckResult> VerifyEndpointIsReadyForNativeTimeouts(EndpointI
 
                     try
                     {
-                        channel.ExchangeDeclarePassive("nsb.delay-delivery");
```

Comment:
I think we might want to add something to this section to make it aware of v1 vs v2 and log the appropriate problem.